src/ripple/nodestore/impl/Shard.h

```diff
@@ -37,7 +37,7 @@ namespace NodeStore {
 using PCache = TaggedCache<uint256, NodeObject>;
```

Comment:
Missing `#include <ripple/nodestore/DatabaseShard.h>`.  Nonunity won't build without it.